packages/web-forms/playwright.config.ts

```diff
@@ -102,7 +102,7 @@ export default defineConfig({
 		 * Serve dev mode: for testing with demo/fixtures
 		 */
 		{
-			command: 'yarn vite dev',
+			command: 'yarn dev',
```

Comment:
Why has this changed?

Reply:
The `yarn dev` is the same as the `yarn vite dev`
Here, we are making it run the same command we have in package.json. It alines with [the workflow below](https://github.com/getodk/web-forms/blob/1942dcba9059d508c4bc767542a08225593aae83/packages/web-forms/playwright.config.ts#L115) that also uses the command defined in the package.json.